### DISCUSSION TITLE
Newsfeed feature design

### DISCUSSION THREAD

**Discussion by enderv** [CAWSCIT/ca-meeting-finder-issue-tracker#20] [Ideas]
From 10/14/23 App call

Request for a way to display news items in the app: upcoming events, flyers at world level.

Write up a design for getting this into the app for the next app meeting.

copied from: https://github.com/CAWSCIT/caws-api/issues/98

**Comment by KalobTaulien** [comment]
We may want to have a quick call to discuss the business logic. Some immediate thoughts are:

* Who can post this information? (Certain people or does a committee manage this?)
* How is it displayed in the app? 
* How do we determine what's important and what isn't? A convention on a different continent might not be applicable to all app users. 
* Is there a content review process? 

I think the architecture of a "post" and turning that into an API response would be quite simple. But on that topic:
* Where are images stored? 
* Does that increase our billing?
